source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -489,13 +490,13 @@ def culling_journeys(resp, request):
 
     """
     Why aggregating journeys before culling journeys?
-    We have encountered severe slowness when combining max_nb_journeys(ex: 20) and a big timeframe_duration(ex: 86400s). 
-    It turned out that, with this configuration, kraken will return a lot of journeys(ex: 100 journeys) and the 
+    We have encountered severe slowness when combining max_nb_journeys(ex: 20) and a big timeframe_duration(ex: 86400s).
+    It turned out that, with this configuration, kraken will return a lot of journeys(ex: 100 journeys) and the
     algorithm was trying to figure out the best solution over 5.35E+20 possible combinations
     ( 5.35E+20=Combination(100,20) )!!
-     
+
     aggregated_journeys will group journeys that are similar('similar' is defined by 'Journeys that have the same sequence
```

Comment:
minor: would be nice to shorten this line on the way

Reply:
if it's fine with Black, it's fine with me :) :black_heart:

Reply:
I've never said that we should not be questioning and improving our process, I do think we clearly should, and your first comment never was about our process.

My point was that your comment is irrelevant in this PR as I'm not going to make cosmetic changes when it is the linter's job. The point of having black is to avoid this kind of comment.

It's fine to disagree with the linter's policy, but if yo do, please, fill a change in Black's rules so we can discuss it.